.github/workflows/edFi.admin.dataaccess pullrequest.yml

```diff
@@ -3,8 +3,12 @@ name: EdFi.Admin.DataAccess Pull request build and test
 on:
   workflow_dispatch:
   pull_request:
-    branches: 
+    branches:
       - main-6x
+    paths:
+      - Application/EdFi.Admin.DataAccess/**/*
+      - Application/EdFi.Admin.DataAccess.UnitTests/**/*
+      - Application/EdFi.Common/**/*
```

Comment:
@JBrenesSimpat I didn't think of this when reviewing your ticket: we only need to run this action when files in one of these three directories have changed. Please take the same approach with the Security. DataAccess.

@semalaiappan you might want to think about doing the same thing in the `main` branch. In limiting the action this way, a pull request only runs the jobs that are truly required - and that means we'll get feedback on the pull request more quickly.

Reply:
a small adjustment here, the path for the unit test should be `- test/EdFi.Admin.DataAccess.UnitTests/**/*`

Reply:
Done